src/shared/ui/Modal/Modal.tsx

```
@@ -62,23 +62,29 @@ export function Modal({
         animate={MODAL_MOTION.animate}
         exit={MODAL_MOTION.exit}
         transition={MODAL_MOTION.transition}
-        className={cn('fixed inset-0 z-50 flex w-full items-center justify-center', className)}
+        className={cn('fixed inset-0 z-50 flex w-full items-center justify-center')}
       >
         <div className="absolute inset-0 bg-black/50" onClick={handleOutsideClick} />
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)